README.md

```diff
@@ -36,7 +36,7 @@ Use the following to cite this repository:
 
 ```
 @misc{2021_nasa_prog_models,
-    author    = {Christopher Teubert and Matteo Corbetta and Chetan Kulkarni and Katelyn Jarvis and Matthew Daigle},
+    author    = {Christopher Teubert, Matteo Corbetta, Chetan Kulkarni, Katelyn Jarvis, and Matthew Daigle},
```

Comment:
BibTeX requires the 'and' separator between authors. See https://www.bibtex.com/f/author-field/